frontend/libs/table/src/lib/table/table-display/table-display.component.ts

```diff
@@ -16,13 +14,15 @@ import { Game } from '@dig-kick/models';
 export class TableDisplayComponent {
   random = Math.floor(Math.random() * 4) + 1;
 
-  tableName = input<string>();
+  tableId = input<string>();
 
-  game = derivedAsync<Game>(() =>
-    this._mqttService
-      .observe(`table/${this.tableName()}/game`)
-      .pipe(map((value) => JSON.parse(value.payload.toString()) as Game)),
-  );
+  readonly gameStore = inject(GameStore);
 
-  constructor(private _mqttService: MqttService) {}
+  game = computed<Game | undefined>(() => {
+    const tableId = this.tableId();
+    if (tableId) {
+      return this.gameStore.entityMap()[tableId];
```

Comment:
Does the library provide an entityMap automatically? It is not defined in game.store

Reply:
Yes this is automagically provided: https://ngrx.io/guide/signals/signal-store/entity-management#withentities-feature